app/config/variables.php

```diff
@@ -510,6 +510,70 @@
                 'required' => false,
                 'question' => '',
             ],
+            [
+                'name' => '_APP_STORAGE_BACKBLAZE_ACCESS_KEY',
+                'description' => 'Backblaze access key. Required when the storage adapter is set to Backblaze. Your Backblaze keyID will be your access key. You can get your keyID from your Backblaze console.',
+                'introduction' => '0.13.0',
```

Comment:
```suggestion
                'introduction' => '0.14.0',
```

Maybe even 0.14.2? Not sure

Reply:
Let's update this for all new vars.

Reply:
I have addressed all of these in the latest PR, which has been merged now by torsten. So closing this PR. Thanks for the review tho😊